library/src/rtc_stockham_gen.cpp

```
@@ -315,9 +343,50 @@ std::string stockham_rtc(const StockhamGeneratorSpecs& specs,
+            reg2lds = std::make_unique<Function>(kernel->generate_lds_from_reg_output_function());
+            device  = std::make_unique<Function>(kernel->generate_device_function());
+            break;
+        }
+        case PPT_SBRR:
+        {
+            auto kernel_pp = static_cast<StockhamPartialPassKernelRR*>(kernel.get());
+
+            lds2reg = std::make_unique<Function>(kernel_pp->generate_lds_to_reg_input_function());
+            reg2lds
+                = std::make_unique<Function>(kernel_pp->generate_lds_from_reg_output_function());
+            lds2reg_pp_steps = std::make_unique<Function>(
+                kernel_pp->generate_lds_to_reg_input_step_1_2_function());
+            reg2lds_pp_steps = std::make_unique<Function>(
+                kernel_pp->generate_lds_from_reg_output_pp_step_1_2_function());
+            twiddle_multiply_pp = std::make_unique<Function>(
+                kernel_pp->generate_twiddle_multiply_pp_function(direction));
+            device = std::make_unique<Function>(kernel_pp->generate_device_function());
+            break;
+        }
+        case PPT_SBCC:
+        {
+            auto kernel_pp = static_cast<StockhamPartialPassKernelCC*>(kernel.get());
+
+            lds2reg
+                = std::make_unique<Function>(kernel_pp->generate_lds_to_reg_input_pp_function());
+            reg2lds
+                = std::make_unique<Function>(kernel_pp->generate_lds_from_reg_output_pp_function());
+            lds2reg_pp_steps = std::make_unique<Function>(
+                kernel_pp->generate_lds_to_reg_input_step_3_4_function());
+            reg2lds_pp_steps = std::make_unique<Function>(
+                kernel_pp->generate_lds_from_reg_output_pp_step_3_4_function());
+            local_transpose_pp
+                = std::make_unique<Function>(kernel_pp->generate_local_transpose_pp_function());
+            device = std::make_unique<Function>(kernel_pp->generate_device_function());
+            break;
+        }
```

default case should raise an exception

---

Thanks, new commit should address this.